### DISCUSSION TITLE
Add an "Crash Player" Feature for Toxic Griefer

### DISCUSSION THREAD

**Discussion by ImNotHeree** [YimMenu/YimMenu#2094] [Ideas]
I was in a lobby. There was a griefer who was constantly killing others and he was kicked a thousand times but he always rejoined. A "crash" feature would delay or even stop this. what do you think?

**Comment by maybegreat48** [comment]
> he was kicked a thousand times but he always rejoined

You can block his join in the player database

**Reply by ImNotHeree** [reply]
oh. your right nvm.. thank you
